css-two-column-layout/styles.css

```diff
@@ -0,0 +1,129 @@
+.container {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  justify-content: right;
+}
+
+.column-full {
+  flex-basis: 100%;
+  justify-content: center;
+  text-align: center;
```

Comment:
Don't pollute your layout classes. `column-` classes should only include `flex-basis` or `width` and nothing else. Please apply these additional declarations to a separate class that you can add in addition to the `column` class